tests/baselines/reference/user/fp-ts.log

```diff
@@ -0,0 +1,913 @@
+Exit Code: 1
+Standard output:
+test/Applicative.ts(6,1): error TS2593: Cannot find name 'describe'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha` and then add `jest` or `mocha` to the types field in your tsconfig.
```

Comment:
@sandersn do we need to alter this test to exclude a `test` directory or something? or include a `types` field?

Reply:
yes, I remember having to do this with other projects. Can you open a bug for it?